Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -70,13 +76,13 @@ class AutocompleteTextField: UITextField, UITextFieldDelegate {
     }
 
     func textField(textField: UITextField, shouldChangeCharactersInRange range: NSRange, replacementString string: String) -> Bool {
+        var allowChractersChange = true
         if completionActive && string.isEmpty {
             // Characters are being deleted, so clear the autocompletion, but don't change the text.
-            removeCompletion()
-            return false
+            allowChractersChange = false
         }
-
-        return true
+        removeCompletion()
```

Comment:
I don't think we want to remove the completion every time a character is entered. This causes the text to flicker every time a character is entered.